pkg/cli/admin/release/extract.go

```diff
@@ -349,6 +349,28 @@ func (o *ExtractOptions) Run(ctx context.Context) error {
 		}
 	}
 
+	// versionInImageConfig stores the version from the label of the image
+	// At the moment it is used later only for the logging purpose and thus is not blocking if failures occur upon getting its value
```

Comment:
If the errors are not blocking then maybe they should be logged as warnings, not errors?